back-end/model/ResetToken.js

```diff
@@ -58,11 +58,15 @@ export const createResetToken = async (user) => {
   });
 };
 
+//--------------VALIDATE-RESET-TOKEN--------------\\
```

Comment:
what is this :D

Reply:
die validateResetToken function gehört zum setpassword in der server.js und ist zum überprüfen, ob der resettoken noch gültig ist